test/cmd/txsim/cli.go

```diff
@@ -177,7 +177,7 @@ well funded account that can act as the master account. The command runs until a
 				opts.SuppressLogs()
 			}
 
-			encCfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+			encCfg := encoding.MakeTestConfig(app.ModuleEncodingRegisters...)
```

Comment:
noticed cli test failing due to not having `AccountI` interface registered, feels weird using test config function here but I think it is the correct encoding.

Reply:
txsim is a CLI testing tool, correct?